modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/model_impl.ftl

```diff
@@ -944,12 +944,8 @@ public class ${entity.name}ModelImpl extends BaseModelImpl<${entity.name}> imple
 				_setModifiedDate = true;
 			</#if>
 
-			<#if entityColumn.isOrderColumn() && columnBitmaskEnabled>
-				_columnBitmask = -1L;
-			</#if>
-
 			<#if entityColumn.isFinderPath() || (validator.isNotNull(parentPKColumn) && (parentPKColumn.name == entityColumn.name))>
-				<#if !entityColumn.isOrderColumn() && columnBitmaskEnabled>
+				<#if columnBitmaskEnabled>
```

Comment:
@brianchandotcom 
Just want to highlight, I did not make this change only for 7.3, but I think this change should be safe to backport to all of the versions (7.0, 7.1, 7.2) currently in maintain, cause the reason we made this is a commit back in Nov. 2012 (https://github.com/liferay/liferay-portal/commit/abd8ee820587fd9c6506232d1d6824f6767b6a66).

Please correct me if I was wrong.

Thanks.
Tina.

Tina.